packages/atomic/src/components/common/facets/category-facet/value-link.spec.ts

```diff
@@ -0,0 +1,107 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {createTestI18n} from '@/vitest-utils/testing-helpers/i18n-utils';
+import {html} from 'lit';
+import {vi, describe, it, expect, beforeAll} from 'vitest';
+import {
+  renderCategoryFacetValueLink,
+  type CategoryFacetValueLinkProps,
+} from './value-link';
+
+describe('renderCategoryFacetValueLink', () => {
+  let i18n: Awaited<ReturnType<typeof createTestI18n>>;
+
+  beforeAll(async () => {
+    i18n = await createTestI18n();
+  });
+
+  const renderComponent = (
+    props: Partial<CategoryFacetValueLinkProps> = {},
+    children = html`<div>Child</div>`
+  ) => {
+    const defaultProps: CategoryFacetValueLinkProps = {
+      displayValue: 'Test Value',
+      numberOfResults: 42,
+      i18n,
+      onClick: vi.fn(),
+      isParent: false,
+      isSelected: false,
+      searchQuery: '',
+      isLeafValue: false,
+      setRef: vi.fn(),
+      children: html`<div>Child</div>`,
+    };
+    const mergedProps = {...defaultProps, ...props};
+    return renderFunctionFixture(
+      html`${renderCategoryFacetValueLink({props: mergedProps})(children)}`
+    );
+  };
+
+  it('should render the facet value link with correct structure', async () => {
+    const container = await renderComponent();
+    const listItem = container.querySelector('li');
+    const button = container.querySelector('button');
+
+    expect(listItem).toBeInTheDocument();
+    expect(button).toBeInTheDocument();
+    expect(button).toHaveAttribute('aria-pressed', 'false');
+  });
+
+  it('should render as parent when isParent is true', async () => {
```

Comment:
For each of the boolean props we're testing, we should also test when the value is false.